mgradm/cmd/support/config/extractor.go

```diff
@@ -86,13 +88,17 @@ func extract(globalFlags *types.GlobalFlags, flags *configFlags, cmd *cobra.Comm
 
 	// Pack it all into a tarball
 	log.Info().Msg(L("Preparing the tarball"))
-	tarball, err := utils.NewTarGz(flags.Output)
+
+	supportFileName := getSupportConfigFileSaveName()
+	supportFilePath := fmt.Sprintf("%s/%s.tar.gz", flags.Output, supportFileName)
```

Comment:
```suggestion
	supportFilePath := path.Join(flags.Output, supportFileName)
```
Using `path.Join()` is better. Adding the `tar.gz` suffix to the result of `getSupportConfigFileSaveName()` would be more consistent.

Reply:
the variable `supportFileName` is also used to define the path for files inside the tar file. If we add the extension to that it will also land inside the tar file and the extract folder will have it on the name. This was the reason I added the extension outside

Reply:
I will change the code to use path.join

Reply:
> the variable `supportFileName` is also used to define the path for files inside the tar file. If we add the extension to that it will also land inside the tar file and the extract folder will have it on the name. This was the reason I added the extension outside

OK, I oversaw this, thanks for explaining

Reply:
changed